test/specs/scenarios/document.js

```diff
@@ -141,5 +142,27 @@ export const documentScenarios = async (lang) => {
       uploadFileAndClickConfirmButton(documentUpload, confirm, 'face.jpeg')
       verificationComplete.verifyUIElements(copy)
     })
+
+    // it('should verify accessibility for the uploader screen', async () => {
+    //   goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}`)
+    //   runAccessibilityTest(driver)
+    // })
+
+    // it('should verify accessibility for the document upload confirmation screen', async () => {
+    //   goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}`)
+  
+    //   documentUpload.getUploadInput()
+    //   documentUpload.upload('passport.jpg')
+    //   runAccessibilityTest(driver)
+    // })
```

Comment:
Would advise adding a TODO or FIXME comment to remember/inform why these tests were commented out